egov-indexer/open-hearing-indexer.yml

```
@@ -48,3 +48,9 @@ ServiceMaps:
                 outJsonPath: $.Data.hearingDetails.hearingType
               - inJsonPath: $.caseFilingDate
                 outJsonPath: $.Data.hearingDetails.caseFilingDate
+              - inJsonPath: $.statusOrder
+                outJsonPath: $.Data.hearingDetails.statusOrder
+              - inJsonPath: $.hearingTypeOrder
+                outJsonPath: $.Data.hearingDetails.hearingTypeOrder
+              - inJsonPath: $.hearingDurationInMillis
+                outJsonPath: $.Data.hearingDetails.hearingDurationInMillis
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)